content/en/docs/setup/production-environment/dockershim.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Dockershim Deprecation FAQ
+content_type: concept
+aliases: [ '/dockershim']
+weight: 40
+---
+
+This document goes over some frequently asked questions and their answers. For
+more detail on the deprecation of Docker as a container runtime for Kubernetes
+kubelets, and what that means, check out the blog post
+[Don't Panic: Kubernetes and Docker](/blog/2020/12/02/dont-panic-kubernetes-and-docker/)
+
+### Can I still use Docker in Kubernetes 1.20?
+
+Yes, the only thing changing in 1.20 is a single warning log printed at startup
```

Comment:
should we mention that it's in kubelet startup, specifically? (and maybe link to the docs for kubelet?)

https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/